components/navigation/Navigation.tsx

```diff
@@ -37,12 +37,12 @@ const Navigation = () => {
           <Navbar.Collapse>
             {/* pushes the buttons to the right */}
             <Nav as="ul" className="ms-auto"></Nav>
-            <Button variant="dark" href="/">
+            <Button href="/" variant="dark" className="px-2 px-sm-3">
               <FaMagnifyingGlass className="me-0 me-sm-2" />
               <span className="d-none d-sm-inline-block">Search</span>
             </Button>
-            <Button href="/join" className="ms-2">
-              <FaBoltLightning className="d-none d-md-inline-block" />
+            <Button href="/join" className="px-2 px-sm-3 ms-1 ms-sm-2 ms-lg-3 ">
+              <FaBoltLightning className="d-none d-sm-inline-block" />
```

Comment:
All seems fine to me, some of these changes will make it look very slightly different from Josh's version but I suspect it's not actually important just tiny layout differences.